tests/console/yast2_samba.pm

```diff
@@ -0,0 +1,279 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use strict;
+use base "consoletest";
+use testapi;
+
+
+
+sub run() {
+    select_console 'root-console';
+
+    # check network at first
+    assert_script_run("if ! systemctl -q is-active network; then systemctl -q start network; fi");
+    # install local ldap server
+    assert_script_run("/usr/bin/zypper -n -q in yast2-auth-server yast2-ldap openldap2 openldap2-client krb5-server krb5-client tdb-tools");
+
+    script_run("/sbin/yast2 auth-server; echo yast2-auth-server-status-\$? > /dev/$serialdev", 0);
+    # check ldap server configuration started
+    assert_screen 'yast2_ldap_configuration_startup';
+    send_key 'alt-f';
+    assert_screen 'yast2_ldap_configuration_general-setting_firewall';
+    sleep 1;
```

Comment:
can you please check all sleep and replace them. sleeps are very unreliable. See https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/1346 for a good example. Every sleep should be replaceable by either nothing, `assert_screen`, `wait_still_screen`, `wait_serial` or `wait_screen_change`